### PR TITLE
Updated Onboarding

### PR DESCRIPTION
**Pull request by ndkazu**
In this first step, tests are disabled, and will be upgraded next.
Assets status name have been updated, and a new call which changes status after positive investor's vote has been added

### CODE REVIEW

File: pallets/onboarding/src/types.rs [hunk 1]
```diff
@@ -15,10 +15,12 @@ pub enum AssetStatus {
 	EDITING,
 	REVIEWING,
 	VOTING,
-	FINALIZING,
+	ONBOARDED,
```
Comment (letodunc) [inline pallets/onboarding/src/types.rs:18]: There should be some test update for these changes. It looks like there is no test coverage on them

**Review by letodunc — CHANGES_REQUESTED** [review]

File: pallets/onboarding/src/types.rs (hunk 1 above)
Comment (letodunc) [reply pallets/onboarding/src/types.rs:18]: I just reviewed the issue commentary about the tests. Why not updating the tests in this issue?
Those changes seem to be those to be done in issue [129](https://github.com/Fair-Squares/fair-squares/issues/129). Is this issue merged with the current one?

File: pallets/onboarding/src/lib.rs [hunk 2]
```diff
@@ -340,7 +340,7 @@ pub mod pallet {
 			);
 			let asset = Self::houses(collection_id.clone(), item_id.clone()).unwrap();
 			let status = asset.status;
-			ensure!(status == AssetStatus::VOTING, Error::<T>::VoteNedeed);
+			ensure!(status == AssetStatus::FINALIZED, Error::<T>::VoteNedeed);
```
Comment (ilhanu) [inline pallets/onboarding/src/lib.rs:343]: This status change is the final change that a house in an acquisition flow acquires. Should it not check before that the OFFERED status is currently the case ?

**Review by ilhanu — APPROVED** [review]

File: pallets/onboarding/src/lib.rs (hunk 2 above)
Comment (ndkazu) [reply pallets/onboarding/src/lib.rs:343]: This line is in the `do_buy` function which execute the final transaction. At this point of the workflow, the asset has been already `Finalized` by the notary.

**Review by letodunc — APPROVED** [review]

**Pull request merged**